algebra/tests/quat/various.c

```diff
@@ -0,0 +1,60 @@
+/*
+ * Phoenix-Pilot
+ *
+ * Unit tests for matrix library
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Piotr Nieciecki
+ *
+ * This file is part of Phoenix-Pilot software
+ *
+ * %LICENSE%
+ */
+
+#include <unity_fixture.h>
+
+#include <quat.h>
+
+
+/* Quaternions used for tests */
+
+/* Matrix without zero and one */
+static const quat_t V1 = { .a = 2.0f, .i = 2.0f, .j = 2.0f, .k = 2.0f };
```

Comment:
We previously talk about vectors, where `V` is better, but here lets use `Q`. This might come handy, when you will test functions operating both on vectors and quaternions
```suggestion
static const quat_t Q1 = { .a = 2.0f, .i = 2.0f, .j = 2.0f, .k = 2.0f };
```